pkg/staticpod/staticpod.go

```diff
@@ -103,6 +103,15 @@ func Run(dir string, args Args) error {
 		return err
 	}
 
+	// Check to make sure we aren't double mounting directories and the files in those directories
+	for _, dir := range args.Dirs {
+		for _, file := range files {
+			if strings.HasPrefix(file, dir) {
+				logrus.Warnf("file %s is being double mounted with directory %s", file, dir)
```

Comment:
Nit:
```suggestion
				logrus.Warnf("Mount of file %s shadows content of mounted directory %s", file, dir)
```

Reply:
In most cases this isn't going to be a warning that the user can actually do anything about. Instead of warning, do we maybe just want to remove the explicit file mount if it is covered by a directory?

Similarly, do we need to check for directories mounted under existing directories?